_includes/tags.html

```diff
@@ -19,19 +19,19 @@
                        using the literal string '/blog/'. Path is used to
                        create the filtered URL: {{ path }}?filter_tags={{ tag }}.
 
-   display_header:     The optional boolean flag to display the tags_header element.
-                       Defaults to True. Adds class 'tags__hide-header' if falsy
-                       value passed (e.g., '', False, None).
+   hide_header:        The optional boolean flag to hide the tags_header element.
+                       Defaults to False. Adds class 'tags__hide-header' if True
+                       value passed (e.g., '', True, None).
```

Comment:
Yep, it should default to False if None is passed. Im going to change it to (e.g., '', True, False) which is better example of what is expected.